.github/workflows/ci-stability-release-branches.yaml

```diff
@@ -0,0 +1,108 @@
+name: ci-stability-release-branches
+
+on:
+  schedule:
+    - cron: "0 19 * * *" # 11:00 AM PST / 20:00 CET / 03:00 CST
+    - cron: "0 21 * * *" # 01:00 PM PST / 22:00 CET / 05:00 CST
+    - cron: "0 23 * * *" # 03:00 PM PST / 00:00 CET / 07:00 CST (next day CET)
+    - cron: "0 1 * * *"  # 05:00 PM PST / 02:00 CET / 09:00 CST (next day CET)
+  workflow_dispatch:
+    inputs:
+      release_branch:
+        description: The release branch to process
+        required: true
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ format('{0}-{1}', github.workflow, github.event_name) }}
+
+jobs:
+  fetch-release-branches:
```

Comment:
WDYT about: `get-active-branches`?

Reply:
but it's not active branches, because we are excluding master